data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
+  Each small molecule was run alone in solution and docked to Mpro to compute absolute free energies of binding.
+  Simulations were run using GROMACS-5.0.4 or GROMACS-2020 and are stored as compressed binary XTC files.
```

Comment:
PDB files defining the topology in the XTC files are critical to these. Can you be sure to show how to retrieve those before you show how to retrieve the XTCs? Also, do they include solute only, solute and ions, all atoms?